terminal/src/main/java/org/jline/terminal/impl/DumbTerminalProvider.java

```diff
@@ -46,8 +43,8 @@ public Terminal sysTerminal(
                 systemStream,
                 name,
                 type,
-                new FileInputStream(FileDescriptor.in),
-                new FileOutputStream(systemStream == SystemStream.Error ? FileDescriptor.err : FileDescriptor.out),
+                System.in,
+                systemStream == SystemStream.Error ? System.err : System.out,
```

Comment:
I'm not sure about those lines.
The provider is required to create a system terminal, which is supposed to be backed by the real terminal emulator where the JVM is running.  If someone changes `System.out` or `System.err`, there's no guarantee about the behavior. If the goal is to be able to run tests, you need to initialize a terminal and set it on the `AnsiConsole` before calling `systemInstall`.

Reply:
right, undone this